Triangulation/examples/Triangulation/triangulation.cpp

```diff
@@ -36,8 +36,8 @@ int main()
     std::cout << "There are " << edges.size()
               << " vertices on the convex hull." << std::endl;
 
-#include "triangulation1.cpp" // See below
-#include "triangulation2.cpp"
+#include "triangulation1.h" // See below
+#include "triangulation2.h"
```

Comment:
Probably should also be done in doc/Triangulation/examples.txt see: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.0-Ic-13/logs_master/Triangulation.log

Reply:
Ah, so that was the reason for those 2 strange files!

Reply:
It should be sections of an example file.

See [`\snippet`](https://www.doxygen.nl/manual/commands.html#cmdsnippet) in Doxygen documentation.